crates/bevy_reflect/src/lib.rs

```diff
@@ -549,6 +549,8 @@ mod reflect;
 mod reflectable;
 mod remote;
 mod set;
+mod short_names;
+pub use short_names::ShortName;
```

Comment:
Nit: Can we move this export so it's with the other exports (and yes, the mods and exports are a bit all over the place in this file haha)?